sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonController.java

```diff
@@ -80,13 +80,23 @@ public TitleLayout getPersonViewTitleLayout(PersonDto personDto) {
 	}
 
 	public void selectOrCreatePerson(final PersonDto person, String infoText, Consumer<PersonReferenceDto> resultConsumer, boolean saveNewPerson) {
+		selectOrCreatePerson(person, infoText, resultConsumer, saveNewPerson, false, null);
+	}
+
+	public void selectOrCreatePerson(
+		final PersonDto person,
+		String infoText,
+		Consumer<PersonReferenceDto> resultConsumer,
+		boolean saveNewPerson,
+		boolean showWithoutMatches,
+		String infoTextWithoutMatches) {
 		// This builds a selection field for all potential similar persons if any.
 		// The user can choose to merge or create a new person in case there is a similar person in the system.
```

Comment:
These comments (as well as the ones further below) seem to be outdated and don't match the method logic. 
With this large amount of parameters, a Javadoc comment explaining the logic would be appropriate.